libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -715,6 +715,8 @@ public static String getEventNameForStat(AnalyticsTracker.Stat stat) {
             case NOTIFICATION_LIKED:
             case NOTIFICATION_QUICK_ACTIONS_LIKED:
                 return "notifications_comment_liked";
+            case NOTIFICATION_QUICK_ACTIONS_QUICKACTION_TOUCH:
+                return "quick_action_touch";
```

Comment:
Same of above, should we use "quick_action_touched"?

Reply:
👍 addressed in 64a386c062 as well